src/mscorlib/src/System/Delegate.cs

```diff
@@ -179,7 +179,7 @@ public override int GetHashCode()
             else
                 return unchecked((int)((long)this._methodPtrAux));
             */
-            return GetType().GetHashCode();
+            return ( _target != null ? _target.GetHashCode() : 0) * 33 + GetType.GetHashCode();
```

Comment:
I don't really understand this comment; you mean instead of GetType().GetHashCode()?